### ISSUE TITLE
Add editor warnings for nodes that are not properly initialized

### ISSUE THREAD

**Issue by NathanLovato** [GDquest/godot-metroidvania-2d#90]
Use tool mode and `_get_configuration_warning` to display warnings on nodes that lack a proper setup. e.g. unset nodepath or invalid nodepath.

**Comment by guilhermehto** [comment]
I started playing around with this, it seems to be a bit... buggy.

Added this:
```gdscript
func _get_configuration_warning() -> String:
	return "Actor Path must be defined.\n" if _actor_path.is_empty() else ""
```

to `PassThroughDetector.gd`.

If you unset `_actor_path`'s value and save, the warning _won't_ appear. Though, if you make any changes to the code i.e. add a blank line and save, the warning _will_ appear. The same is not true if you, for instance, remove the `CollisionShape` child of this same node: as soon as you save the game, the warning will appear. And yes, the script is already in `tool` mode.

I'll keep working on it and put up a draft PR so we can keep discussing.
